apiserver/facades/client/storage/register.go

```diff
@@ -22,8 +22,32 @@ import (
 // Register is called to expose a package of facades onto a given registry.
 func Register(registry facade.FacadeRegistry) {
 	registry.MustRegister("Storage", 6, func(ctx facade.Context) (facade.Facade, error) {
-		return newStorageAPI(ctx) // modify Remove to support force and maxWait; add DetachStorage to support force and maxWait.
-	}, reflect.TypeOf((*StorageAPI)(nil)))
+		return newStorageAPIV6(ctx) // modify Remove to support force and maxWait; add DetachStorage to support force and maxWait.
+	}, reflect.TypeOf((*StorageAPIv6)(nil)))
+
+	registry.MustRegister("Storage", 7, func(ctx facade.Context) (facade.Facade, error) {
+		return newStorageAPIV7(ctx) // support force option on import-fileystem.
```

Comment:
The highest facade version (now 7, was 6) just uses the `newStorageAPI` directly.